### DISCUSSION TITLE
CentOS 7.X Latest Slow loading..

### DISCUSSION THREAD

**Discussion by packet33** [antonym/netboot.xyz#1032] [Q&A] [answered]
Hello,

Is anyone else having this issue? I'm trying to use CentOS 7 / Graphical and so far it's been over 20 minutes waiting for the graphical to load. Server is on a gigabit connection but according to the switch its only receiving @ 1mbps. Connected to another server on that same switch i can do a speed test and get about 900/900. What server is it pulling these files from, how is it selected and is it possible to change it?

Thanks,

**Comment by antonym** [comment, marked as answer]
It's using mirror.centos.org so it may be doing a redirect to the "best" mirror that's available. I tried one and it loaded up pretty quick for me.

If you need to do an override, you could hop into the ipxe shell once the menu is loaded and do a:

```
set centos_mirror http://mirror.centos.org
set centos_base_dir centos
```

The mirror settings are loaded from https://boot.netboot.xyz/boot.cfg before the menu is loaded, so you should be able to override that setting and then load the CentOS menu up with those updated settings.
